### ISSUE TITLE
Improve missing credentials error & Setup Default AWS profile via CLI

### ISSUE THREAD

**Issue by DavidWells** [serverless/serverless#1727]
TLDR: http://www.screencast.com/t/1UR8eOMoU (1min 30 sec)

Upgrading from 0.5 to 1.0 causes some weird issues, all relating to the `~/.aws` credentials file containing old account information from v0.5 and expecting the `[default]` aws credentials (described [here](https://github.com/serverless/serverless/blob/v1.0/docs/guide/provider-account-setup.md#setting-a-default-aws-profile) linking to the [setup instructions](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html#cli-config-files)  )

This is something I think we could fix with a `sls setup` command in the CLI to clean up the  `~/.aws` credentials file and setup the `[default]` aws credentials if they don't exist yet via the command prompt.

##### Serverless Framework Version: 
1.0

##### Operating System: 
OSX
##### Suggested Behavior:

![image](https://cloud.githubusercontent.com/assets/532272/17317985/af837dfa-5837-11e6-9c42-0274ef80af9e.png)

##### Actual Behavior:

This error is displayed with no supporting information `ServerlessError: TimeoutError: Missing credentials in config`

Related:
https://github.com/serverless/serverless/issues/1520
https://github.com/serverless/serverless/issues/1632
https://github.com/serverless/serverless/issues/1701
https://github.com/serverless/serverless/issues/1649

**Comment by DavidWells** [comment]
Sending people to read through https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html#cli-config-files is a negative UX. 

Lets remove this point of friction via:

- a setup step via the CLI
- and/or a GUI electron app

Either (or both) lets people plug in the AWS aws_access_key_id and aws_secret_access_key and writes it to their `~/.aws` credentials file if the `[default]` block doesn't already exist.

**Comment by flomotlik** [comment]
We've fixed the error messages and added docs for setting up the AWS credentials properly. Imho this can be considered fixed at the moment. I don't think we should actively set up profiles ourselves and mess with AWS  config files directly.
